Tooltip.jsx

```diff
@@ -3,178 +3,149 @@
  * @flow strict
  */
 
-/**
- * A wrapper around rc-tooltip that code splits off the third party code into a
- * separate bundle. rc-tooltip will be downloaded only if it is needed to render
- * a tooltip. Use this instead of rc-tooltip.
- */
-
 import * as React from "react";
-import invariant from "./tools/invariant";
-import {
-  makeCancelable,
-  type CancelablePromise,
-} from "./tools/cancelablePromise";
-
-type Props = {
-  +children: React.Node,
-  +placement?: string,
-  +defaultVisible?: boolean,
-  +visible?: boolean,
-  +mouseEnterDelay?: number,
-};
-
-// TODO(dmnd): rc-tooltip doesn't have types :(
-type RcTooltipT = React.ComponentType<Props>;
-
-// Module state. We only want to load RcTooltip once, so this state is stored
-// at the module level.
-let RcTooltip: null | RcTooltipT = null;
-let waitForRcTooltipResolve = null;
-let waitForRcTooltipReject = null;
-
-function initModuleState() {
-  return new Promise((resolve, reject) => {
-    waitForRcTooltipResolve = resolve;
-    waitForRcTooltipReject = reject;
-  });
+import {StyleSheet, css} from "aphrodite";
+import TooltipTrigger from "react-popper-tooltip";
+
+import colors from "./colors";
+
+const VERTICAL_OFFSET = 8;
```

Comment:
nit: only use all uppercase if the var is exported (https://github.com/airbnb/javascript#naming--uppercase)
```suggestion
const verticalOffset = 8;
```